package.json

```diff
@@ -19,6 +19,7 @@
     "@bcrs-shared-components/contact-info": "1.0.8",
     "@bcrs-shared-components/corp-type-module": "1.0.3",
     "@bcrs-shared-components/court-order-poa": "1.0.1",
+    "@bcrs-shared-components/date-picker": "1.0.0",
```

Comment:
Certify component version (line 17) should be "1.0.12" per Steph's recent PR. Please rebase and make sure it's correct thereafter.

Reply:
kk, will do